### ISSUE TITLE
feature request - change default profile tab to "notes"

### ISSUE THREAD

**Issue by GretaFrank** [hzrd149/nostrudel#39]
whenever i want to check what i have just recently posted i click on "profile" and am taken to my "about" page, which is literally the same every single day and never changed even once and i don't actually need to see it 20 times a day.

what i really want is to see is the "notes" tab of my "profile" page.

i think "notes" should be the default tab at least for my OWN profile.

when looking at OTHER people's profile, perhaps the "about" tab makes more sense, though "notes" wouldn't be a bad option either as the default.

or with a bit more effort you can just put the "Notes" below the "about" in the same tab so that you can simply scroll down from the "about" to the "notes" ... after all, the person looking for notes will be scrolling ANYWAY.

the point here is to eliminate additional unnecessary step of having to click on profile and then on notes.  there should only be one click, not two.

**Comment by hzrd149** [comment]
I agree that the "about" page is pretty useless now. although I don't think making the "notes" page the default would fix it.
It would be a good solution for those users who only care about notes, but for other users who are just exploring id like to surface other things like articles / streams / communities / lists / emojis when you click on a user.

Ultimately Id like to build something like https://nosta.me where it shows a summary of the users activity, then you can dive into notes or "other stuff" :)
